src/utils/colors.ts

```diff
@@ -16,17 +16,23 @@ export function getColor(index: number) {
     return colors[index % colors.length];
 }
 
-const colorForType: Map<string, string> = new Map<string, string>([
-    ['JSON', 'var(--type-color-json)'],
-    ['GeoJSON', 'var(--type-color-geojson)'],
-    ['CSV', 'var(--type-color-csv)'],
-    ['PDF', 'var(--type-color-pdf)'],
-    ['HTML', 'var(--type-color-html)'],
-]);
-
-export function getColorForResourceType(type: 'JSON' | 'GeoJSON' | 'CSV' | 'PDF' | 'HTML' | undefined) {
-    if (type === undefined) {
-        return;
+export function getColorForResourceType(type?: 'JSON' | 'GeoJSON' | 'CSV' | 'PDF' | 'HTML') {
```

Comment:
Why a switch over a map?

Reply:
Reason was to keep it consistent with `icons.ts`. I would be fine with using maps in both, however.

Reply:
Ah okay, that's fine